composer/cicd_sample/utils/add_dags_to_composer.py

```diff
@@ -52,17 +54,19 @@ def upload_dags_to_composer(dags_directory: str, bucket_name: str) -> None:
 
         for dag in dags:
             # Remove path to temp dir
-            # if/else is a relative directory workaround for our tests
-            current_directory = os.listdir()
-            if "dags" in current_directory:
-                dag = dag.replace(f"{temp_dir}/", "dags/")
-            else:
-                dag = dag.replace(f"{temp_dir}/", "../dags/")
-
-            # Upload to your bucket
-            blob = bucket.blob(dag)
-            blob.upload_from_filename(dag)
-            print(f"File {dag} uploaded to {bucket_name}/{dag}.")
+            dag = dag.replace(f"{temp_dir}/", name_replacement)
+
+            try:
+                # Upload to your bucket
+                blob = bucket.blob(dag)
+                blob.upload_from_filename(dag)
+                print(f"File {dag} uploaded to {bucket_name}/{dag}.")
+            except FileNotFoundError:
+                current_directory = os.listdir()
+                print(
+                    f"dags/ directory not found in {current_directory}, you may need to override the default value of name_replacement to point to a relative directory"
```

Comment:
should this be
```
f"{name_replacement} directory not found in {current_directory}, you may need to override the default value of `name_replacement` to point to a relative directory"
```
otherwise regardless of the value it will always print `dags/ directory not found`, which might confuse future debugging.

Reply:
YES. Excellent call.